docs/source/examples/test.myst

```diff
@@ -167,6 +167,12 @@ from cirq import LineQubit, X, DensityMatrixSimulator
 ```
 
 ```{testcode}
+# TODO: Remove imports (temp fix).
+import numpy as np
+from cirq import Circuit, depolarize
+from cirq import LineQubit, X, DensityMatrixSimulator
```

Comment:
Without this temp fix there is an error that `DensityMatrixSimulator` is not defined, so it is somehow not imported from above. Not sure how to fix. cc @crazy4pi314

(It used to pass because there was a `from cirq import *` in the global docs config.)